packages/html/src/SimpleTable.ts

```diff
@@ -0,0 +1,51 @@
+import { HTMLWidget } from "@hpcc-js/common";
+import { select as d3Select } from "d3-selection";
+
+export class SimpleTable extends HTMLWidget {
+    protected _table;
+    protected _tbody;
+    _transformedData;
```

Comment:
Looks dubious - remove.

Reply:
It's used to allow extending classes to transform `this.data()` before calling update...

Reply:
Where (and why)?

Reply:
In the `BreakdownTable.ts` (it wasn't part of the PR until I amended it this morning)

Reply:
This is not referenced anywhere in this PR, it should be removed (if needed in dependent code, we can review it then).

Reply:
I've replaced it with this method. Does that work? That will allow extending widgets to overwrite that method with their own transformation logic.
```javascript
    transformData(data) {
        return data;
    }
```

Reply:
Add that method in your other PR, when its needed - otherwise your just adding a "Nothing" method.

Reply:
As far as I can tell - even if an extending method overrode it, it has no use in this class?

Reply:
Done, it's removed

Reply:
`_transformedData` is still there...